test/browser/features/international-address-enter-address.feature

```diff
@@ -145,24 +145,13 @@ Feature: International address - Enter your address
     Then they see an error summary with failed validation message: "Region must only include letters a to z, numbers 0 to 9 and .,-\/* or '"
     Then they see an error summary with failed validation message: "Town, suburb or city must only include letters a to z, numbers 0 to 9 and .,-\/* or '"
 
-  Scenario: Change selected international country, selected same country leads to international page
-    Given they have selected the country "Kenya"
-    When they click the continue button
-    Then they should see international address form
-    And they see the change country link "Kenya"
-    When they click change country link
-    Then they should see the country selector page
-    When they click the continue button
-    Then they should see international address form
```

Comment:
I don't think you should remove this test, it was intentional to mimic what we thought was the originally the behaviour at the time, was for the same country to be selected when the link was clicked now the behaviour is that we expect the dropdown not to be selected

Reply:
I suppose the scenario wording needs changing `Change selected international country, unselected country validation error` or some better wording.
The idea is that the last

`Then they should see international address form` changes b'cos now they still on the country selector but a validation error occurs with `Select the country you live in` instead